backend/devices/bitbox02/protobuf.go

```diff
@@ -41,8 +42,8 @@ var btcMsgOutputTypeMap = map[txscript.ScriptClass]messages.BTCOutputType{
 	txscript.WitnessV0ScriptHashTy: messages.BTCOutputType_P2WSH,
 }
 
-var ethMsgCoinMap = map[string]messages.ETHCoin{
-	"eth":                 messages.ETHCoin_ETH,
+var ethMsgCoinMap = map[coin.Code]messages.ETHCoin{
+	coin.CodeETH:          messages.ETHCoin_ETH,
 	"eth-erc20-usdt":      messages.ETHCoin_ETH,
```

Comment:
not sure how to deal with those, but prefer to deal with them later if at all.

Reply:
why not add them as say `coin.CodeERC20USDT`, `coin.CodeERC20Link` and so on?

Reply:
cause conceptually we might add 500 of them, so i'd rather keep the codes in erc20.go (or move to some json file) and change this part instead in some other clever way that avoids tons of lines.